templates/admin/users.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html.twig' %}
+
+{% block main_content %}
+    {% if users is not empty %}
+        <ul>
+            {% for user in users %}
+                <li>
+                    <a href="{{ path('admin_user_dashboard', {userId:user.id}) }}">
+                        {{ user.name }}
+                    </a>
+                </li>
+            {% endfor %}
+        </ul>
+    {% else %}
```

Comment:
Just found about the `else` clause of a `for` :
https://twig.symfony.com/doc/3.x/tags/for.html#the-else-clause